blazor/treegrid/rows/row-template.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: post
+title: Row Template in Blazor Tree Grid Component | Syncfusion
+description: Checkout and learn here all about Row Template in Syncfusion Blazor Tree Grid component and much more details.
+platform: Blazor
+control: Tree Grid
+documentation: ug
+---
+
+# Row Template in Blazor Tree Grid Component
+
+To know about Row Template in Blazor tree grid Component, you can check this video.

```

Comment:
To know more about Row Template in the Blazor tree grid Component, you can check this video.